clang/test/dpct/cusolver_range.cu

```diff
@@ -0,0 +1,18 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.2, cuda-10.0
+// UNSUPPORTED: v8.0, v9.2, v10.0
+// RUN: dpct --format-range=none -out-root %T/cusolver_range %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only
+// RUN: FileCheck --input-file %T/cusolver_range/cusolver_range.dp.cpp --match-full-lines %s
+
+// CHECK: #include <sycl/sycl.hpp>
+// CHECK-NEXT: #include <dpct/dpct.hpp>
+// CHECK-NEXT: #include <dpct/lapack_utils.hpp>
+#include "cusolverDn.h"
+
+void test0() {
+  // CHECK: char range = 'A';
+  cusolverEigRange_t range = CUSOLVER_EIG_RANGE_ALL;
+  // CHECK: range = 'V';
+  range = CUSOLVER_EIG_RANGE_V;
```

Comment:
I think there is mapping in oneMKL.
cusolverEigRange_t => oneapi::mkl::rangev
CUSOLVER_EIG_RANGE_ALL => oneapi::mkl::rangev::all
CUSOLVER_EIG_RANGE_V => oneapi::mkl::rangev::values
CUSOLVER_EIG_RANGE_I => oneapi::mkl::rangev::indices